### ISSUE TITLE
Error inflating BottomNavigationView

### ISSUE THREAD

**Issue by rubo** [xamarin/AndroidSupportComponents#27]
### Xamarin.Android Version (eg: 6.0):
7.0.2.37

### Operating System & Version (eg: Mac OSX 10.11):
Windows 10

### Support Libraries Version (eg: 23.3.0):
25.0.1-rc1

### Describe your Issue:
It throws InflateException when there's android.support.design.widget.BottomNavigationView anywhere in layout files.

### Steps to Reproduce (with link to sample solution if possible):
Put the following in any layout file:
```xml
<android.support.design.widget.BottomNavigationView
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:design="http://schema.android.com/apk/res/android.support.design"
     android:id="@+id/navigation"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_gravity="start"
     design:menu="@menu/nav" />
```
or simpler, just:
```xml
<android.support.design.widget.BottomNavigationView />
```

### Include any relevant Exception Stack traces, build logs, adb logs:
`Android.Views.InflateException: Binary XML file line #1: Error inflating class android.support.design.widget.BottomNavigationView ---> Java.Lang.Reflect.InvocationTargetException: Exception of type 'Java.Lang.Reflect.InvocationTargetException' was thrown. ---> Java.Lang.NoClassDefFoundError: Failed resolution of: Landroid/support/transition/AutoTransition; ---> Java.Lang.ClassNotFoundException: Didn't find class "android.support.transition.AutoTransition" on path:`

**Comment by UKDeveloper99** [comment]
+1 Having the exact same issue.

**Comment by UKDeveloper99** [comment]
Update: 
I managed to get it working by installing the Xamarin Android Support Transition nuget.
I also installed the Xamarin Android Support Fragment nuget, that may be unrelated but my project was crashing without it.

All my nuget android support libraries are on the pre release version 25.0.1-rc1.

I also installed the latest version of Xamarin Studio and Xamarin.Android from the Alpha channel. Check for updates then go to the alpha channel.
